tests/core_tests/block_validation.h

```diff
@@ -230,3 +230,37 @@ struct get_test_options<gen_block_low_coinbase> {
     hard_forks, 0
   };
 };
+
+struct gen_block_miner_tx_out_has_no_view_tag_before_hf_view_tags : public gen_block_accepted_base<2>
```

Comment:
Maybe a matter of taste but I would probably balk when my mind came up with such a long name, and start to search for something shorter :)

Reply:
Forget this comment - I saw now that this is for tests, and because there are so many of them, and such specific ones, you need long names.